scripts/filter_language_issues.py

```
@@ -0,0 +1,181 @@
+    # Ignore comma/parenthesis whitespace in code blocks
+    if rule_id == "COMMA_PARENTHESIS_WHITESPACE":
+        return True
+
+    # Ignore very short text snippets (likely code)
+    if len(text.strip()) <= 3:
+        return True
+
+    return False
+
+
+def check_text_filtered(text: str, tool, file_path: str = "") -> list:
```

Ruff (0.12.2)

Missing type annotation for function argument `tool`

(ANN001)